Themis.xcodeproj/project.pbxproj

```diff
@@ -1874,6 +1887,30 @@
 			defaultConfigurationName = Release;
 		};
 /* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		6D0A527C25FA2E7000C49E55 /* XCRemoteSwiftPackageReference "openssl-apple" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/julepka/openssl-apple";
+			requirement = {
+				kind = upToNextMajorVersion;
+				minimumVersion = 1.1.1080202;
```

Comment:
```suggestion
				minimumVersion = 1.1.108203;
```

TODO: re-define version after merging https://github.com/cossacklabs/openssl-apple/pull/22

Reply:
yeah, and the link will point to the cosacklabs repo.